### PR TITLE
Describe login as a customer from the admin panel

### PR DESCRIPTION
**Pull request by ZhovnirMaria**
## Purpose of this pull request

This pull request describes login as a customer from the admin panel

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [x] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [ ] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

https://docs.magento.com/user-guide/customers/login-as-customer.html

### CODE REVIEW

**Comment by erikmarr** [comment]
Hi @ZhovnirMaria thanks for your contribution! How is your new section different from the "Customer account permission for remote shopping assistance" section in this same topic? I might be missing something but they both seem to be describing how a merchant can login as the customer to see what the customer sees.

**Comment by erikmarr** [comment]
@magento give me 2.4.0 instance

File: src/customers/login-as-customer.md [hunk 1]
```diff
@@ -62,6 +62,18 @@ Enabling Login as Customer requires that you enable the feature in your Magento
...
 
+## Login as a customer from the admin panel
+
+1. On the _Admin_ sidebar, go to **Customers** > _All Customers_.
+
+1. Open a user in edit mode.
+
+1. In the **Customer Information** panel, choose the **Account Information** section.
+
+1. Set the **Allow remote shopping assistance** toggle to `Yes`.
```
Comment (erikmarr) [inline src/customers/login-as-customer.md:73]:
```suggestion
1. Set the **Allow remote shopping assistance** to `Yes`.
```

File: src/customers/login-as-customer.md [hunk 2]
```diff
@@ -62,6 +62,18 @@ Enabling Login as Customer requires that you enable the feature in your Magento
...
+
+   After all these steps, the administrator can log in as a user without his permission from the storefront.
```
Comment (erikmarr) [inline src/customers/login-as-customer.md:75]:
```suggestion
   The administrator can now log in as a user without their permission from the storefront.
```

**Review by erikmarr — APPROVED** [review]

**Comment by erikmarr** [comment]
@magento import code to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@erikmarr the branch with code successfully imported into`magento-devdocs/merchdocs` repository. Branch name: `imported-magento-merchdocs-1324`.

**Pull request merged**